cmake/Platform/Linux/LYWrappers_linux.cmake

```diff
@@ -55,16 +55,31 @@ endfunction()
 function(ly_apply_debug_strip_options target)
 
     find_program(GNU_STRIP_TOOL strip)
-    if (NOT GNU_STRIP_TOOL OR NOT ${LY_STRIP_DEBUG_SYMBOLS})
+    if (NOT GNU_STRIP_TOOL)
         return()
     endif()
 
```

Comment:
I'd recommend putting the find_program inside the below if statement since its not necessary to find it unless we are stripping symbols, right?  Save some config time.
In addition, I'd also add a find for objcopy and show a warning if either are missing.  Or even an error and tell people to supply them or turn off LY_STRIP_DEBUG_SYMBOLS

Reply:
It is here because the gnu strip tool is still used if the option is set to false. The step to detach the debug symbol still relies on strip to remove the debug symbols after it has been extracted out. I will remove the check here, there is already a check on top of this file and i will set the results from that check here to prevent this function if either strip or objcopy cannot be found